site/zenodo_rdm/curation/jobs.py

```diff
@@ -30,6 +28,6 @@ def build_task_arguments(cls, job_obj, since=None, **kwargs):
         if since is None and job_obj.last_runs["success"]:
             since = job_obj.last_runs["success"].started_at
         else:
-            since = since or datetime.utcnow()
+            since = since or datetime.now(timezone.utc)
 
-        return {"since": str(since)}
+        return {"since": since}
```

Comment:
I might have misunderstood the goal of normalising "since" parameter, but I thought we aimed to remove this return from all the child JobType class methods - and the idea was to return it by default in the equivalent private method, ping @ntarocco for confirmation

Reply:
> I might have misunderstood the goal of normalising "since" parameter, but I thought we aimed to remove this return from all the child JobType class methods - and the idea was to return it by default in the equivalent private method, ping @ntarocco for confirmation

The goal was not to remove entirely the `since`, but only the casting to str. If, for some reason, the job requires the `since` param (because it has to compute some dates or be aware of it), then the job implementation will return it and inject in the celery task.

@palkerecsenyi it looks that [it is needed](https://github.com/zenodo/zenodo-rdm/blob/3627edef32c64b314a92dc81ab9c0bd69b2054f4/site/zenodo_rdm/curation/tasks.py#L107)!